content/blog/call-for-hpe-ezmeral-unified-analytics-beta-testers.md

```diff
@@ -0,0 +1,29 @@
+---
+The HPE Ezmeral team has been hard at work creating the next generation analytics and advanced analytics platform. To give you an early look at what we’ve been up to, we have created the HPE Ezmeral Early Access beta program. Through our HPE Ezmeral Early Access beta program, you can start working with the tools and frameworks packaged inside HPE Ezmeral Unified Analytics. Your feedback will be critical in helping to shape our final product.
+
+**What’s new with HPE Ezmeral Unified Analytics?**
+
+The new HPE Ezmeral Unified Analytics is a packaged offering that contains best-of-breed open source tools, including data analytics tools like Apache Spark and Presto, data visualization tools like Apache SuperSet, data engineering tools like Airflow and Great Expectations, and data science tools like Kubeflow, MLFlow, and Feast. These open source packages are generally better than any proprietary solutions on the market and are also fully supported by HPE as part of HPE Ezmeral Unified Analytics. HPE’s open source packages are also pre-integrated. Integration can be one of the biggest challenges of consuming open source solutions. Our work at the data/feature layer radically simplifies your overall solution without compromising the benefits of using 100% open source API’s. In addition to framework integration, we have also spent time securing these components out-of-the-box, removing yet another headache.
+
+HPE Ezmeral Unified Analytics is a hybrid and multi-cloud solution. This is a fancy way of saying that, at the time of GA (general availability), HPE Ezmeral Unified Analytics will run everywhere you want it to run. It can be installed on-premises, at the edge, or in a public cloud. We will provide our next-gen HPE Ezmeral Unified Analytics product in an appliance-like form factor that can be installed and ready to use in under 10 minutes. The built-in global namespace will allow you to easily connect multiple unified analytics workloads and data fabrics to share compute and data across different clouds and different workflows.
+
+**Let your opinion be known!**
+
+One could consider HPE Ezmeral Unified Analytics as an opinionated set of software. Part of the purpose of this beta is to ensure that our opinion makes sense. We are trying to select the most important open source products in data engineering, data analytics, and data science. Because we don’t expect our customers to choose all of our tools, we designed our workflows and pipelines to allow easy replacement of an included component by an alternative. We try to provide a comprehensive selection of products, but we also try to stay out of your way if you have preferences. For example, while we include Super Set for building reports and dashboards, if you prefer to use Tableau, our JDBC Presto API’s are available for a quick swap. If you prefer using Weights and Bias’, you can swap out our ML Flow registry solution. Models can be published to Kserve or Sagemaker Endpoints.
```

Comment:
For example, while we include Super Set for building....

Previous occurrence of Apache Superset was in one word (SuperSet).
Maybe for consistency purpose, spell it the same throughout the post.

Same remark for "you can swap out our ML Flow registry solution". Is it "ML Flow" or "MLFlow" registry ? Please confirm.

Reply:
Apache Superset and MLflow... I figured out how to change it